WebDriverAgentLib/Categories/XCUIElement+FBTap.m

```diff
@@ -26,13 +27,35 @@ @implementation XCUIElement (FBTap)
 
 - (BOOL)fb_tapWithError:(NSError **)error
 {
+  if (SYSTEM_VERSION_GREATER_THAN_OR_EQUAL_TO(@"10.0") && [self fb_hasInvertedParentWindow]) {
+    // This indicates the fact, that the element is not affected by XCTest landscape bug and
+    // is properly clicked by calling the native tap selector
+    [self tap];
```

Comment:
I tried, but this is real black magic to me how it works internally and in which way these bloody coordinates are corrupted by XCTest internals. After spending hours on investigation I decided to stay with this approach, because it's proven to work.